### ISSUE TITLE
Differences between Olmo2 1B config file and .sh file used for pretraining

### ISSUE THREAD

**Issue by baon6052** [allenai/OLMo#867]
### ❓ The question

Hi!

I believe there might be a few differences between the olmo2 1B pretraining [config file](https://github.com/allenai/OLMo/blob/main/configs/official-0425/OLMo2-1B-stage1.yaml) and the actual [.sh file](https://github.com/allenai/OLMo/blob/bf536fdfb5ab9b77c8defac2d7ca37db05eea733/scripts/augusta/peteish1-anneal.sh) executed for the experiments as noted [here](https://github.com/allenai/OLMo/issues/865#issuecomment-3079843611).

- Learning rate: [yaml](https://github.com/allenai/OLMo/blob/main/configs/official-0425/OLMo2-1B-stage1.yaml#L51) says `4.0e-4` while [.sh file](https://github.com/allenai/OLMo/blob/bf536fdfb5ab9b77c8defac2d7ca37db05eea733/scripts/augusta/peteish1-anneal.sh#L92) says `0.000074487` 
- Flash attention: [yaml](https://github.com/allenai/OLMo/blob/bf536fdfb5ab9b77c8defac2d7ca37db05eea733/scripts/augusta/peteish1-anneal.sh#L88) says `true` while [.sh file](https://github.com/allenai/OLMo/blob/bf536fdfb5ab9b77c8defac2d7ca37db05eea733/scripts/augusta/peteish1-anneal.sh#L88) says `false`

Would you please be able to clarify? I would like to replicate the exact pretraining environment.

Thanks!

**Comment by baon6052** [comment]
Hi @baileykuehl !

Thank you for your comment and looking into this. It looks like `peteish1.sh` inherits it's config from [configs/peteish1-google.yaml](https://github.com/allenai/OLMo/blob/main/configs/peteish1-google.yaml) according to line [66](https://github.com/allenai/OLMo/blob/main/scripts/augusta/peteish1.sh#L66)? 

If this is the case, is [configs/peteish1-google.yaml](https://github.com/allenai/OLMo/blob/main/configs/peteish1-google.yaml) the actual 1B pretraining config and not [configs/official-0425/OLMo2-1B-stage1.yaml](https://github.com/allenai/OLMo/blob/main/configs/official-0425/OLMo2-1B-stage1.yaml)?

**Comment by baileykuehl** [comment]
No problem - feel free open another ticket if you run into any other issues!
